### DISCUSSION TITLE
Should we release 0.66.5?

### DISCUSSION THREAD

**Discussion by lunaleaps** [reactwg/react-native-releases#8] [Patches]
# 0.67.0 has been released
Please direct future discussion to [here](https://github.com/reactwg/react-native-releases/discussions/10)
---
Current Release: [0.66.4](https://github.com/facebook/react-native/releases/tag/v0.66.4)

Conversations on this thread are limited to 0.66 releases major issues and backport (cherry-pick) requests from commits that are already on master.

An example of a good such request is a bug fix for a serious issue that has been merged into main but did not make the 0.66.4 cut, with a link to the specific commit hash on main with the commit to cherry-pick, like this example link: [facebook/react-native@bd2b7d6](https://github.com/facebook/react-native/commit/20b0eba581a00e5e7e300f6377379b836617c147)

In other words, if you cannot point to a particular commit on main, then your request likely belongs as a new issue.

### List of cherry picks
1.
### Local commits to backport to main
1.

**Comment by billnbell** [comment]
Yes release 0.66.5 to fix the issue https://github.com/reactwg/react-native-releases/discussions/6

**Comment by xiao99xiao** [comment]
@lunaleaps This may be a little off-topic but please don't use https://github.com/facebook/react-native/commit/bd2b7d6 as the example, because it is a typical bad commit that ruined the onDismiss feature for several times because of being merged here and there, we took months across several releases and with this commit https://github.com/facebook/react-native/commit/d85d5d2e1974b463318e4c86da29a5ccdd60a977 to end the nightmare eventually.

**Reply by kelset** [reply]
good point, I've changed it to another example commit - thanks for noticing!

**Comment by billnbell** [comment]
OK can we release?

**Reply by lunaleaps** [reply]
Since we aim to release 67 in the next week or so, I'd rather focus our fixes and attention to that

**Comment by kelset** [comment]
In case we are going to consider a new patch for 0.66, this is one we might want to cherry pick https://github.com/facebook/react-native/commit/6f126740fa560d7a831979b9f3747baacfb28dba to solve this https://github.com/facebook/react-native/issues/32463 (as pointed out by @cortinico in the last comment)

**Reply by cortinico** [reply]
Agree. With 0.67.x behind the corner, I don't think it's extremely crucial. But should we ship 0.66.6, having that included would be great 👍

**Reply by kelset** [reply]
yeah exactly, I'm just writing it for future memory

**Comment by tido64** [comment]
If we're doing 0.66.5, can we cherry-pick https://github.com/facebook/react-native/commit/60cef850bd3fd12c32ee1196bd19a559592d1465? iOS doesn't build if you run `pod install --project-directory=ios`.
